src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/FileUploadWithTagComponent.tsx

```diff
@@ -0,0 +1,812 @@
+/* eslint-disable react-perf/jsx-no-new-object-as-prop */
```

Comment:
This file is way too big, and will be difficult to maintain - can you please split it into several smaller files instead? There are several `render*` helper functions within this component. Those would make good candidates to separate into their own files.

- [ ] Split file into smaller more manageable parts

Reply:
I have now refactored and split out EditWindow and FileList to their own components. Also dragged out shared functions for this and the old FileUpload component.